packages/react-charting/src/components/SankeyChart/SankeyChart.types.ts

```diff
@@ -113,3 +118,29 @@ export interface ISankeyChartStyles {
 
   calloutContentRoot?: IStyle;
 }
+
+/**
+ * Set of strings which are used when renering the chart.
+ */
+export interface ISankeyChartStrings {
```

Comment:
rename to ISankeyChartAccessibilityProps

Reply:
Some of the strings are normal strings and not accessibility-specific. Should we be separating the two kinds of strings into different collections?

Reply:
you can make put it along with other base props calling it linkDescription.

Reply:
I chose to split it across two different collections. Our team would like to make some modifications to how values are rendered (e.g. Showing values such as "360K" instead of "359453") and I expect to add such as utility functions on `ISankeyChartStrings`. But I plan to do that as a separate PR.